custom/icds_reports/views.py

```diff
@@ -1712,7 +1712,6 @@ def get(self, request, *args, **kwargs):
         dump = DishaDump(state_name, query_month)
         return dump.get_export_as_http_response(request)
 
-    @property
-    @quickcache([])
```

Comment:
oh ho! i will revert this if thats the case

How does quickcache work on instance? in a different request, an object of class will not fetch this?

Reply:
quickcache works the same on an instance as off - it varies only by the arguments passed in.  Since there are no `vary_on` arguments (it's an empty list), this will hit the same cache every time.  If you needed to vary it by some property of the instance, you could use a `vary_on` parameter of say, `['self.domain']`.  Looks like this method doesn't need that though, since it doesn't take parameters.  It'll return the cached value until it's invalidated or expires (after 5 minutes, by default).

If you're interested in reading more, I wrote some docs on the topic here:
https://github.com/dimagi/commcare-hq/blob/master/docs/caching_and_memoization.rst